### PR TITLE
Changed showPicture function

### PR DESCRIPTION
**Pull request by erinlimbo**
changed showPicture function to be able to specify specific pictures. Updated index.html to account for this

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -19,27 +19,11 @@ var projectList = [
...
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
-}
-
 /** 
  * Displays/hides portrait picture.
```
Comment (laskdaf) [inline portfolio/src/main/webapp/script.js:23]: update docs to include param annotation.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -24,9 +26,20 @@
...
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Erin!</h1>");
-  }
+    private List<String> names;    
+
+    @Override
+    public void init() {
+        names = new ArrayList<>();
```
Comment (laskdaf) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: nit: consider declaring the arraylist with default values
eg.
`new ArrayList<String>(Arrays.asList("Erin", "Ruchi", "Owen"));`

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (laskdaf) [inline portfolio/src/main/webapp/script.js:24]: update docs to include param annotation.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 2 above)
Comment (erinlimbo) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: done.

**Review by laskdaf — APPROVED** [review]

**Pull request merged**